server/go.mod

```diff
@@ -121,6 +119,7 @@ require (
 	github.com/google/uuid v1.3.0 // indirect
 	github.com/gopherjs/gopherjs v1.17.2 // indirect
 	github.com/gorilla/css v1.0.0 // indirect
+	github.com/graph-gophers/graphql-go v1.5.1-0.20230110080634-edea822f558a // indirect
```

Comment:
Do we still depend on this?

Reply:
Yep, seems like it's an indirect dependency now. I did do a `modules-tidy`.